src/Umbraco.Web.UI.Client/src/packages/core/workspace/components/workspace-split-view/workspace-split-view-variant-selector.element.ts

```diff
@@ -488,7 +488,7 @@ export class UmbWorkspaceSplitViewVariantSelectorElement<
 				: html`
 						<uui-button
 							class="split-view"
-							label="Open Split view for ${variant.language.name}"
+							label=${`${this.localize.term('buttons_openSplitView')} ${variant.language.name}`}
```

Comment:
In this case we can use a function in the dicationary. If you change you openSplitView key to a method then you can pass the language name as an argument. It will look something like this:

{
  openSlitViewFor: (languageName) => (languageName ? `Open split view for '${languageName}'` : 'Open split view'),
}

Reply:
Great 🎉